CONTRIBUTING.md

```diff
@@ -112,11 +109,7 @@ with a patch attached for review.
 
 #### Waiting For Feedback
 
-It may take a while for committers to review. Please be patient during this
-time as all committers are volunteers on the project. If a significant amount
-of time has lapsed since your submission, such as a couple of months, feel free
-to either update your BZ, PR, or email the dev list with a message to bump your
-issue. Sometimes things get lost in all the work and we need a reminder :smile:
+It may take a while for committers to review. Please be patient during this time as all committers are volunteers on the project. If a significant amount of time has elapsed since your submission, such as a couple of months, feel free to either update your BZ, PR or email the dev list with a message to bump your issue. Sometimes things get lost in all the work and we need a reminder 😄
 
```

Comment:
I think :smile: should be retained here.
As above, please retain the original line length limits.